ros2param/ros2param/verb/list.py

```diff
@@ -87,7 +87,7 @@ def main(self, *, args):  # noqa: D102
 
             # wait for all responses
             for future in futures.values():
-                rclpy.spin_until_future_complete(node, future)
+                rclpy.spin_until_future_complete(node, future, timeout_sec=1.0)
```

Comment:
not sure just ignoring timeout is good enough here. i would add check the FutureReturnCode, and if gets TIMEOUT, notify the user what node's parameter cannot be queried. i would like to hear from others.

Reply:
I agree, if there is a problem communicating with one of the parameter servers then we should warn the user, but I think we are getting some feedback with the "exception" printed at L104.